TezosKit/Client/PocketNetworkClient.swift

```diff
@@ -0,0 +1,101 @@
+// Copyright Keefer Taylor, 2019
+
+import Foundation
+import PocketSwift
+
+public class PocketNetworkClient: NetworkClient {
+    
+    /// Network name
+    private let network = "TEZOS"
+    /// Network ID
+    private let netID: String
+    /// Developer ID
+    private let devID: String
+    /// Pocket Instance that will relay requests
+    private let pocket: Pocket
+    /// Headers which will be added to every request.
+    private let headers: [Header]
+    /// A response handler for RPCs.
+    private let responseHandler: RPCResponseHandler
+    /// The queue that callbacks from requests will be made on.
+    internal let callbackQueue: DispatchQueue
+    /// Initialize a new AbstractNetworkClient.
+    /// - Parameters:
+    ///   - devID: Pocket developer identifier.
+    ///   - netID: Tezos network identifier.
+    ///   - headers: The path to the remote node.
+    ///   - urlSession: The URLSession that will manage network requests.
+    ///   - headers: Headers which will be added to every request.
+    ///   - callbackQueue: A dispatch queue that callbacks will be made on.
+    ///   - responseHandler: An object which will handle responses.
+    public init(
+        devID: String,
+        netID: String,
+        headers: [Header] = [],
+        callbackQueue: DispatchQueue,
+        responseHandler: RPCResponseHandler
+        ) {
+        self.netID = netID
+        self.devID = devID
+        self.headers = headers
+        self.callbackQueue = callbackQueue
+        self.responseHandler = responseHandler
+        self.pocket = Pocket(devID: devID, network: network, netID: netID, maxNodes: 10, requestTimeOut: 10_000)
```

Comment:
optional: I'd consider making 10 and 10_000 be parameters in the initalizer defaulted to these values.